client/lib/css-hot-reload/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import io from 'socket.io-client';
+
+/**
+ * Internal dependencies
+ */
+import { cssBuildFailed, cssBuilding } from 'state/application/actions';
+import i18n from 'lib/mixins/i18n';
+
+// https://developer.mozilla.org/en/docs/Web/HTML/Element/link
+var standardAttributes = [
+	'crossorigin',
+	'href',
+	'hreflang',
+	'media',
+	'rel',
+	'sizes',
+	'title',
+	'type'
+];
+
+/**
+ * @returns {object} 
+ */
+function bustHashForHrefs( { name, oldValue } ) {
+    const value = 'href' === name
+        ? `${ oldValue.split( '?' ).shift() }?v=${ new Date().getTime() }`
```

Comment:
wasn't there a comment here before? if not, it might be useful

```js
// http://some.site.com/and/a/path?with=a&query -> http://some.site.com/and/a/path?v=13508135781
```